exporter/parquetexporter/README.md

```diff
@@ -0,0 +1,28 @@
+# Parquet File Exporter
+
+Sends pipeline data to Parquet files.
```

Comment:
For a subsequent PR, document the file's format.

Reply:
The Parquet file format will in all likelihood live under opentelemetry-proto - here is the PR for the schema here: https://github.com/open-telemetry/opentelemetry-proto/pull/346

Reply:
That's OK, just link it here then so that users of this component know where to get this info.